File: java/oop/InterfaceApp/src/com/techlabs/icrudable/InvoiceDb.java
package com.techlabs.icrudable;

public class InvoiceDb implements Icrudable {
	@Override
	public void create() {
		System.out.println("Invoice's create operation");
		
	}

	@Override
	public void read() {
		System.out.println("Invoice's create operation");
	}

	@Override
	public void update() {
		System.out.println("Invoice's update operation");
		
	}

	@Override
	public void remove() {
		System.out.println("Invoice's remove method");	
	}

}

File: dotNet/csharp/InterfaceCrudApp/InterfaceCrudApp/InvoiceDb.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace InterfaceCrudApp
{
    class InvoiceDb : Icrudable
    {
        public void Create()
        {
            Console.WriteLine("Invoice's Create operations........");
        }

        public void Read()
        {
            Console.WriteLine("Invoice's Read operations........");
        }

        public void Remove()
        {
            Console.WriteLine("Invoice's Remove operations........");
        }

        public void Update()
        {
            Console.WriteLine("Invoice's Update operations........");
        }
    }
}
